src/modals/ImageModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import {createPortal} from 'react-dom';
+import {SFC} from 'types';
+import {useState} from 'react';
+import * as S from './Styles';
+import DefaultAvatar from '../../assets/default-avatar.svg';
```

Comment:
I don't think we need to do absolute path here:
<img width="379" alt="image" src="https://github.com/thenewboston-developers/thenewboston-Frontend/assets/52558124/6045cf10-35d6-47b7-89f4-963a1034c456">